examples/tutorials/tutorial_RANS_xv15.py

```diff
@@ -0,0 +1,101 @@
+import math
+
+import flow360 as fl
+from flow360.examples import TutorailRANSXv15
+
+fl.Env.preprod.active()
+
+TutorailRANSXv15.get_files()
+project = fl.Project.from_file(
+    TutorailRANSXv15.mesh_filename,
+    name="Tutorial Time-accurate RANS CFD on XV-15 from Python",
+)
+volume_mesh = project.volume_mesh
+
+with fl.SI_unit_system:
+    rotation_cylinder = volume_mesh["innerRotating"]
```

Comment:
```suggestion
    rotation_zone = volume_mesh["innerRotating"]
```
To not be confused with the cylinder entity concept. It is a zone and then happens to be cylindrical.

Reply:
OK, updated